ICSharpCode.Decompiler/Documentation/XmlDocLoader.cs

```diff
@@ -135,7 +147,7 @@ static string LookupLocalizedXmlDoc(string fileName)
 			return null;
 		}
 
-		static string GetLocalizedName(string fileName, string language)
+		private static string GetLocalizedName(string fileName, string language)
 		{
 			string localizedXmlDocFile = Path.GetDirectoryName(fileName);
 			localizedXmlDocFile = Path.Combine(localizedXmlDocFile, language);
```

Comment:
Can we merge these two calls to Path.Combine?

Reply:
I intentionally kept everything "as is" except for the lookup logic

Reply:
Please?

Reply:
Done